packages/legend-application-studio/src/stores/editor-state/element-editor-state/ClassState.ts

```diff
@@ -364,6 +368,25 @@ export class ClassState {
     }
   }
 
+  moveColumn(sourceIndex: number, targetIndex: number): void {
```

Comment:
This method changes the graph, so it needs to be an `action`, so it's fair. But we centralize graph modification in `DomainGraphModifierHelper`, let's move this there and rename this to `class_rearrangeProperties()`